docs/_layouts/ds6/default.html

```diff
@@ -1,17 +1,67 @@
 <!doctype html>
 <html lang="en">
     <head>
-        <title>eBay skin DS6</title>
+        <title>eBay Skin — for DS6</title>
         <meta name="viewport" content="width=device-width, initial-scale=1"/>
         <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
         <link type="text/css" rel="stylesheet" href="../static/ds6/skin.css"/>
         <link type="text/css" rel="stylesheet" href="../static/ds6/docs.css"/>
     </head>
     <body>
+        <header>
+            <h1>
+                <span class="inline-svg">
```

Comment:
I believe this span needs `role="img"` and `aria-label="eBay"`.

I think `aria-hidden` needs to be `aria-hidden="true"`, but could be wrong...

Reply:
I'll add the role and the label. Also, I'll confirm the aria-hidden.